### DISCUSSION TITLE
Vertical divider: only centered position?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1574] [Elements (button, image, label, loader, ...)] [answered]
About [vertical divider](https://fomantic-ui.com/elements/divider.html#vertical-divider) I read this:

_Due to a change in W3C implementation of absolutely positioned elements in flex containers vertical dividers now currently only support 50/50 splits automatically, and only if not positioned as direct children of flex containers (like grid)._

So, how to have a divider that is not centered? Something like this:

![download (1)](https://user-images.githubusercontent.com/5399265/87422779-86e5be80-c5d9-11ea-8f38-9c7eaf0e3067.png)

Is something that could be reached using [grid](https://fomantic-ui.com/collections/grid.html) or is there a better way?

**Comment by TheJltres** [comment]
Hello,

I think the best implementation for this is border: https://jsfiddle.net/TheJltres/tL8f6kws/4/
If you want to go with only two columns you will have to play with column wide: https://fomantic-ui.com/collections/grid.html#column-widths

Main problem is that you can't have text

**Comment by ko2in** [comment, marked as answer]
If you want to use divider element within grid. Better add customization inside your [divider.overrides](https://github.com/fomantic/Fomantic-UI/blob/master/src/themes/default/elements/divider.overrides). The below CSS will work:
```
.ui.grid > .column > .ui.divider {
  left: auto;
  right: 0;
  -webkit-transform: translateX(50%);
  transform: translateX(50%);
}
```
Here's fiddle (https://jsfiddle.net/2hfwdq6k/). You can add divider element within your grid column. You will need additional customization for responsive layouts.

If you just want the bordered only, FUI already support horizontal segments with grid. Here's fiddle (https://jsfiddle.net/d73zs0gk/).

Or you can use custom CSS to apply border as @TheJltres suggested.
